public/assets/jquery-ui/droppable.js

```diff
@@ -38,7 +38,7 @@ function(e){"function"==typeof define&&define.amd?define(["jquery","./widget"],e
ed))),o.axis&&"y"===o.axis||(e.pageX-t(a).scrollLeft()<o.scrollSensitivity?r=t(a).scrollLeft(t(a).scrollLeft()-o.scrollSpeed):t(window).width()-(e.pageX-t(a).scrollLeft())<o.scrollSensitivity&&(r=t(a).scrollLeft(t(a).scrollLeft()+o.scrollSpeed)))),r!==!1&&t.ui.ddmanager&&!o.dropBehaviour&&t.ui.ddmanager.prepareOffsets(n,e)}}),t.ui.plugin.add("draggable","snap",{start:function(e,i,n){var o=n.options;n.snapElements=[],t(o.snap.constructor!==String?o.snap.items||":data(ui-draggable)":o.snap).each(function(){var e=t(this),i=e.offset();this!==n.element[0]&&n.snapElements.push({item:this,width:e.outerWidth(),height:e.outerHeight(),top:i.top,left:i.left})})},drag:function(e,i,n){var o,r,s,a,l,u,c,h,p,d,f=n.options,m=f.snapTolerance,g=i.offset.left,y=g+n.helperProportions.width,v=i.offset.top,b=v+n.helperProportions.height;for(p=n.snapElements.length-1;p>=0;p--)l=n.snapElements[p].left-n.margins.left,u=l+n.snapElements[p].width,c=n.snapElements[p].top-n.margins.top,h=c+n.snapElements[p].height,l-m>y||g>u+m||c-m>b||v>h+m||!t.contains(n.snapElements[p].item.ownerDocument,n.snapElements[p].item)?(n.snapElements[p].snapping&&n.options.snap.release&&n.options.snap.release.call(n.element,e,t.extend(n._uiHash(),{snapItem:n.snapElements[p].item})),n.snapElements[p].snapping=!1):("inner"!==f.snapMode&&(o=Math.abs(c-b)<=m,r=Math.abs(h-v)<=m,s=Math.abs(l-y)<=m,a=Math.abs(u-g)<=m,o&&(i.position.top=n._convertPositionTo("relative",{top:c-n.helperProportions.height,left:0}).top),r&&(i.position.top=n._convertPositionTo("relative",{top:h,left:0}).top),s&&(i.position.left=n._convertPositionTo("relative",{top:0,left:l-n.helperProportions.width}).left),a&&(i.position.left=n._convertPositionTo("relative",{top:0,left:u}).left)),d=o||r||s||a,"outer"!==f.snapMode&&(o=Math.abs(c-v)<=m,r=Math.abs(h-b)<=m,s=Math.abs(l-g)<=m,a=Math.abs(u-y)<=m,o&&(i.position.top=n._convertPositionTo("relative",{top:c,left:0}).top),r&&(i.position.top=n._convertPositionTo("relative",{top:h-n.helperProportions.height,left:0}).top),s&&(i.position.left=n._convertPositionTo("relative",{top:0,left:l}).left),a&&(i.position.left=n._convertPositionTo("relative",{top:0,left:u-n.helperProportions.width}).left)),!n.snapElements[p].snapping&&(o||r||s||a||d)&&n.options.snap.snap&&n.options.snap.snap.call(n.element,e,t.extend(n._uiHash(),{snapItem:n.snapElements[p].item})),n.snapElements[p].snapping=o||r||s||a||d)}}),t.ui.plugin.add("draggable","stack",{start:function(e,i,n){var o,r=n.options,s=t.makeArray(t(r.stack)).sort(function(e,i){return(parseInt(t(e).css("zIndex"),10)||0)-(parseInt(t(i).css("zIndex"),10)||0)});s.length&&(o=parseInt(t(s[0]).css("zIndex"),10)||0,t(s).each(function(e){t(this).css("zIndex",o+e)}),this.css("zIndex",o+s.length))}}),t.ui.plugin.add("draggable","zIndex",{start:function(e,i,n){var o=t(i.helper),r=n.options;o.css("zIndex")&&(r._zIndex=o.css("zIndex")),o.css("zIndex",r.zIndex)},stop:function(e,i,n){var o=n.options;o._zIndex&&t(i.helper).css("zIndex",o._zIndex)}}),t.ui.draggable}),/*!
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Confusing use of '!'.
* Too many errors. (80% scanned).

<!-- Comment by [SideCI](https://sideci.com) -->